src/main/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/SpecimenMapper.java

```diff
@@ -4,11 +4,11 @@
 import org.springframework.stereotype.Component;
 import uk.nhs.digital.nhsconnect.lab.results.model.edifact.Message;
 
-import java.util.Optional;
+import java.util.List;
 
 @Component
 public class SpecimenMapper {
-    public Optional<Specimen> map(final Message message) {
-        return Optional.empty();
+    public List<Specimen> mapToSpecimens(final Message message) {
```

Comment:
Removed `Optional` from these resource mappers.
From looking at the pathology docs, I haven't seen anything that states they can be optional